cnf-certification-test/accesscontrol/suite.go

```diff
@@ -611,3 +618,14 @@ func Test1337UIDs(env *provider.TestEnvironment) {
 
 	testhelper.AddTestResultLog("Non-compliant", badPods, tnf.ClaimFilePrintf, ginkgo.Fail)
 }
+
+// a test for security context that are allowed from the documentation of the cnf
+// an allowed one will pass the test
+func testContainerSCC(env *provider.TestEnvironment) {
+	var badContainer []string
+	for _, pod := range env.Pods {
+		containersBad := securitycontextcontainer.CheckPod(pod)
```

Comment:
securitycontextcontainer.CheckPod(pod) should return a full container name including namespace and pod name, not just the container name as it might be the duplicated.

Reply:
This is returning the pods that do not fit in a category, Instead we need the list of pods that do not fit SCC category1. In addition to the list of pods not in category 1, calculate the category for the whole CNF and save it in the claim.json file. Would also be nice to store the detailed SCC category for each container (including a dump of the SCC objsct for each)